.github/pull_request_template.md

```diff
@@ -1,29 +1,40 @@
-<!--
-## Tips for pull requests
-* use chat, discussions, etc... to refine an idea for big and/or impactful changes
-* open them relatively early in draft mode and get regular feedback
-* add notes for reviewers and yourself as you go, so it's easy to maintain context
-* refine the pull request message over time
--->
+## Summary
+* what changed and how?
+* why are we changing it?
 
-<!--
-## Finalizing a PR:
+## Details
+* info that couldn't fit into the summary
+* relevant details; tricky parts
+* anything else
 
-### title:
-* reads like a short changelog line
+Fixes #(issue)
 
-### body:
-* describe the current behaviour
-* describe why this particular approach
-* if it's breaking, migration steps
-* note any follow-on work
-
-### content:
-* leave code better than you found it, add docs, tests, etc
--->
+---
+<!-- Note: section break (`---`) onwards is not in CI merge commit -->
 
+## Notes for Reviewers
+* additional context for reviewers
+* maybe specific requests or areas of focus
 
 <!--
-* note any issues that this fixes entirely
-* use `fixes #123 .` to auto-close issues
--->
\ No newline at end of file
+Pull Request(PR) Help
+
+Before Merge Ensure:
+* title reads like a short changelog line entry
```

Comment:
Guidelines for the title should be as close to the title as possible. Possibly as a placeholder text for the title.

Reply:
For pull request templates can't make it a placeholder.

The other thing is that figuring out the title for any nontrivial PR is actually best code to the end, either after you've written the body, it's come out of draft, or it's just shy of approval. That's when everyone knows the most and can nicely summarize it. Placing all that pressure up front is a blocker, though.